docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
   redis:
     image: redis:3.0
   app:
-    build: .
+    image: voxmedia/docker_base_images:ruby_2.2-0.1
```

Comment:
Yup! This image is based on the `ruby:2.2.2` image. The Dockerfile for that is here: https://github.com/voxmedia/docker_base_images/blob/master/ruby/2.2/Dockerfile